### PR TITLE
New article: FWCPv2 access-list rules

### PR DESCRIPTION
**Pull request by DonNateR**
Part of PR #1937 

- Add images

- Incorporate Kelly's edits

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-76e4c6aaad/how-to/).

File: content/dedicated-hosting/firewall-manager-v2-access-list-rules.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+permalink: firewall-manager-v2-access-list-rules/
+audit_date: '2017-03-28'
+title: Firewall Manager v2 - access-list rules
+type: product
```
Comment (stephamon) [inline content/dedicated-hosting/firewall-manager-v2-access-list-rules.md]: Tricksy metadata! Type should be article

File: content/dedicated-hosting/firewall-manager-v2-access-list-rules.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+product_url: dedicated-hosting
+---
+
+Firewall Manager v2 is a new tool within the MyRackspace portal. This article describes the access-list rule feature within this tool.
+
+To learn more about the tool, see [Firewall Manager v2](https://support.rackspace.com/how-to/firewall-manager-v2).
+
+### Prerequisite
+
+Before you use the access-list rules feature, read the Firewall Manager v2 [access-list theory and best practices](https://support.rackspace.com/how-to/firewall-manager-v2-access-list-theory-and-best-practices) article.
```
Comment (stephamon) [inline content/dedicated-hosting/firewall-manager-v2-access-list-rules.md]: Here and throughout, links to other How-To articles don't need the "https://support.rackspace.com" part

File: content/dedicated-hosting/firewall-manager-v2-access-list-rules.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+3. Under **Rules**, click **All Rules**.
+
+    <img src="{% asset_path dedicated-hosting/firewall-manager-v2-access-list-rules/image-rules.png %}" />
+
+   The Rules page shows the rules for the access-list on the selected tab. To view the rules for a different access-list, click the its tab.
+
+### Rackspace internal lines
+
+For Rackspace to manage and monitor your environment properly, we must have some configurations in place on your Cisco firewall. These configurations contain sensitive Rackspace internal data, and the access-list rule that contains them has been hidden from your view in Firewall Manager v2. Because the hidden Rackspace line must remain at the top of your access-list to prevent
```
Comment (stephamon) [inline content/dedicated-hosting/firewall-manager-v2-access-list-rules.md]: Final sentence in the paragraph seems like a few words are missing at the end

File: content/dedicated-hosting/firewall-manager-v2-access-list-rules.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+Following is an example of how the hidden line is indicated in the access-list:
+
+<img src="{% asset_path dedicated-hosting/firewall-manager-v2-access-list-rules/image-redacted-line.png %}" />
+
+### Access-list rule features
+
+The Firewall Manager v2 access-list rules feature has an export function. To use this function, clickthe **Export to .csv** button. You can download the .csv file or open it directly from the next screen prompt.
```
Comment (stephamon) [inline content/dedicated-hosting/firewall-manager-v2-access-list-rules.md]: Need a space between "click" and "the"

File: content/dedicated-hosting/firewall-manager-v2-access-list-rules.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+    <img src="{% asset_path dedicated-hosting/firewall-manager-v2-access-list-rules/image-line-number.png %}" />
+
+4. Specify the line number to move this entry to.
+
+   **Note:** Firewall Manager v2 does not allow you to place rules above the highest hidden Rackspace internal line.
+
+
+### Adding an access-list rule
+
+Before you add an access-list rule, be sure to read and understand all of the best practices documented in the  [Firewall Manager v2 - Theory and Best Practices](https://support.rackspace.com/how-to/firewall-manager-v2-access-list-theory-and-best-practices) article.
```
Comment (stephamon) [inline content/dedicated-hosting/firewall-manager-v2-access-list-rules.md]: For the links to the other Firewall Manager articles, we took out that dash in the title, and used sentence case. So this article is called Firewall Manager v2 access-list theory and best practices. There's another one for this article, and one for the object-groups article below that need the same title edit

**Comment by stephamon** [comment]
👍

**Pull request merged**